src/forms/reset-password-popup/reset-password/tests/index.test.jsx

```diff
@@ -252,4 +291,42 @@ describe('ResetPasswordPage', () => {
     render(reduxWrapper(<IntlResetPasswordPage />));
     expect(store.dispatch).toHaveBeenCalledWith(setCurrentOpenedForm(LOGIN_FORM));
   });
+
+  it('should clear confirmPassword error message when passwords match', async () => {
+    store = mockStore({
+      ...initialState,
+      resetPassword: {
+        status: TOKEN_STATE.VALID,
+        token,
+      },
+    });
+
+    render(reduxWrapper(<IntlResetPasswordPage />));
+    const newPasswordInput = screen.getByLabelText('New password');
+    const confirmPasswordInput = screen.getByTestId('confirmPassword');
+
+    fireEvent.change(newPasswordInput, { target: { value: 'ValidPass123' } });
+    fireEvent.change(confirmPasswordInput, { target: { value: 'ValidPass123' } });
+
+    fireEvent.blur(confirmPasswordInput);
+
+    // Ensure no error message is displayed for confirmPassword
+    expect(screen.queryByText('Confirm your password')).toBeNull();
+  });
+
+  it('should handle mouse down event on submit button', () => {
+    store = mockStore({
+      ...initialState,
+      resetPassword: {
+        status: TOKEN_STATE.VALID,
+      },
+    });
+
+    render(reduxWrapper(<IntlResetPasswordPage />));
+
+    const submitButton = screen.getByRole('button', { name: /Reset password/i, id: 'submit-new-password' });
+    fireEvent.mouseDown(submitButton);
+
+    expect(submitButton).toBeTruthy();
```

Comment:
The ask of unit test is to check that mouseDown event is fired on submit button click but we are not checking that event is fired or not

Reply:
Thank you, updated tests